specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2022-11-01/secrets.json

```diff
@@ -0,0 +1,491 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-11-01",
+    "title": "KeyVaultManagementClient",
+    "description": "The Azure management API provides a RESTful set of web services that interact with Azure Key Vault."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.KeyVault/vaults/{vaultName}/secrets/{secretName}": {
+      "put": {
+        "tags": [
+          "Secrets"
+        ],
+        "operationId": "Secrets_CreateOrUpdate",
+        "description": "Create or update a secret in a key vault in the specified subscription.  NOTE: This API is intended for internal use in ARM deployments. Users should use the data-plane REST service for interaction with vault secrets.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Resource Group to which the vault belongs."
+          },
```

Comment:
You are highly recommended to remove your own definitions of these and use the definitions from common-specification/types/v3/types.json instead

Reply:
This is old existing file, not introduced in this PR, so skipping.